nav_pose_server:
  ros__parameters:
    timeout: 60.0
    position_tolerance: 0.15
    yaw_tolerance: 0.05

nav_pose_controller:
  ros__parameters:
    kp_linear: 1.0
    ki_linear: 0.0
    kd_linear: 0.2
    kp_angular: 1.0
    ki_angular: 0.2
    kd_angular: 0.0
    max_vel_linear: 2.0
    max_vel_angular: 8.0
    tolerance: 0.15